Sources/SpeziLLMOpenAI/LLMOpenAISession.swift

```diff
@@ -7,12 +7,15 @@
 //
 
 import Foundation
-import class OpenAI.OpenAI
+import OpenAPIRuntime
+import OpenAPIURLSession
 import os
 import SpeziChat
 import SpeziLLM
 import SpeziSecureStorage
 
+/// A Swift Logger that logs important information and errors.
+public let logger = Logger(subsystem: "edu.stanford.spezi", category: "SpeziLLMOpenAI")
```

Comment:
I would advise against a global public `Logger` here. Always try to keep it internal and bound to a certain component (e.g. a `struct`)

Reply:
Addressed in: 10fcb63ae15db1c7e219f783654a4e42d1fcd6fc